CHANGELOG.md

```diff
@@ -39,7 +39,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - Renaming files now truncates the filename to not exceed the limit of 255 chars [#2622](https://github.com/JabRef/jabref/issues/2622)
 
 ### Removed
-
+- We removed support for LatexEditor, as it is not under active development. (https://github.com/JabRef/jabref/issues/3199a)
```

Comment:
Please follow the styles from the other entries for creating the URL links. And I think you have an 'a' too much at the issue number

Reply:
yeap. `[#3199](https://github.com/JabRef/jabref/issues/3199)` would be correct.

Reply:
Oops, totally overlooked that :D Should be fixed now, thanks guys